packages/search-adapter/src/empathy/requestors/beacon-tracking.requestor.ts

```diff
@@ -9,20 +9,47 @@ import { Requestor } from '../empathy-adapter.types';
  */
 @injectable()
 export class BeaconTrackingRequestor implements Requestor<TrackingRequest, void> {
+  constructor() {
+    this.detectAdblock()
+      .then(() => (this.isAdblockPresent = false))
+      .catch(() => (this.isAdblockPresent = true));
```

Comment:
Use fat arrow functions, you don't want to return the boolean, you just want to assign it:
```suggestion
      .then(() => {
        this.isAdblockPresent = false;
      })
      .catch(() => {
        this.isAdblockPresent = true;
      });
```